spec/forms/providers/means/regular_outgoings_form_spec.rb

```diff
@@ -0,0 +1,317 @@
+require "rails_helper"
+
+# rubocop:disable Rails/SaveBang
+RSpec.describe Providers::Means::RegularOutgoingsForm do
+  describe "validations" do
+    context "when neither a outgoing type or none are selected" do
+      it "is invalid" do
+        params = { "transaction_type_ids" => [""] }
+
+        form = described_class.new(params)
+
+        expect(form).to be_invalid
+        expect(form.errors).to be_added(:transaction_type_ids, :blank)
+      end
+    end
+
+    context "when at least one outgoing type is selected, but an amount is missing" do
+      it "is invalid" do
+        rent_or_mortgage = create(:transaction_type, :rent_or_mortgage)
```

Comment:
again, don't feel comfortable and don't like this "not very rspec" and inconsistent to codebase way of testing but, ok as it follows the pattern of the previous one.